packages/cli/templates/scaffolder-module/src/actions/example/example.test.ts

```diff
@@ -22,7 +22,7 @@ describe('acme:example', () => {
       logStream: new PassThrough(),
       output: jest.fn(),
       createTemporaryDirectory() {
-        // Usage of mock-fs is recommended for testing of filesystem operations
+        // Usage of createMockDirectory is recommended for testing of filesystem operations
```

Comment:
```suggestion
        // Usage of createMockDirectory from @backstage/backend-test-utils is recommended for filesystem tests
```

Given that this is a change to the published package we'll also need a changeset for this bit. Just highlighting that the scaffolder-module template now recommends usage of this instead of mock-fs.

Reply:
Sure, no problem! Adding one.

Reply:
Added one. Let me know if you want a more detailed description in it.